proc-macros/src/lib.rs

```diff
@@ -84,7 +84,7 @@ pub(crate) mod visitor;
 ///     #[method(name = "bar")]
 ///     fn sync_method(&self) -> String;
 ///
-///     #[subscription(name = "sub", unsub = "unsub", item = "String")]
+///     #[subscription(name = "subscribe", item = "String")]
```

Comment:
I think you need to update the generated code sample below as well right? On line 102

Reply:
nah, it's the actual impl so it's safe.